py/tests/ctm_test.py

```diff
@@ -22,44 +23,66 @@ def ctm_to_blob(ctm, card):
     return pykms.Blob(card, arr);
 
 
-if len(sys.argv) > 1:
-    conn_name = sys.argv[1]
-else:
-    conn_name = ""
+parser = argparse.ArgumentParser(description='Simple alpha blendig test.')
+parser.add_argument('--connector', '-c', dest='connector', default="",
+		    required=False, help='connector to output')
+parser.add_argument('--mode', '-m', dest='modename', default="",
+		    required=False, help='Video mode name to use')
+parser.add_argument('--plane', '-p', dest='plane', default="",
+		    required=False, help='plane number to use')
```

Comment:
You should use type=int, if it's an int. And you should use "default" only if with that you can use the parameter directly. If you'll just later check param == "", then the default doesn't make sense, and you can just check if param was set at all.